profiles/package.mask

```diff
@@ -29,6 +29,10 @@
 
 #--- END OF EXAMPLES ---
 
+# Ilya Tumaykin <itumaykin+gentoo@gmail.com> (18 Jan 2018)
```

Comment:
I will swap your name and email addresses for mine. Hope you don't mind.

LGTM otherwise.

Reply:
Sure. No problem)